src/Mvc/test/Mvc.FunctionalTests/ComponentRenderingFunctionalTests.cs

```diff
@@ -35,59 +36,93 @@ public async Task Renders_BasicComponent()
 
-            AssertComponent("\n    <p>Hello world!</p>\n", "Greetings", content);
+            AssertComponent("\n<p>Hello world!</p>", "Greetings", content);
         }
 
         [Fact]
         public async Task Renders_BasicComponent_UsingRazorComponents_Prerrenderer()
         {
             // Arrange & Act
-            var client = CreateClient(Factory, builder => builder.ConfigureServices(services => services.AddRazorComponents()));
+            var client = CreateClient(Factory
+                .WithWebHostBuilder(builder => builder.ConfigureServices(services => services.AddRazorComponents())));
 
             var response = await client.GetAsync("http://localhost/components");
 
             // Assert
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
             var content = await response.Content.ReadAsStringAsync();
 
-            AssertComponent("\n    <p>Hello world!</p>\n", "Greetings", content);
+            AssertComponent("\n<p>Hello world!</p>", "Greetings", content);
         }
 
         [Fact]
         public async Task Renders_RoutingComponent()
         {
             // Arrange & Act
-            var client = CreateClient(Factory, builder => builder.ConfigureServices(services => services.AddRazorComponents()));
+            var client = CreateClient(Factory.WithWebHostBuilder(builder => builder.ConfigureServices(services => services.AddRazorComponents())));
 
             var response = await client.GetAsync("http://localhost/components/routable");
 
             // Assert
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
             var content = await response.Content.ReadAsStringAsync();
 
-            AssertComponent("\n    Router component\n<p>Routed successfully</p>\n", "Routing", content);
+            AssertComponent("\nRouter component\n<p>Routed successfully</p>", "Routing", content);
         }
 
         [Fact]
         public async Task Renders_RoutingComponent_UsingRazorComponents_Prerrenderer()
         {
             // Arrange & Act
-            var client = CreateClient(Factory, builder => builder.ConfigureServices(services => services.AddRazorComponents()));
+            var client = CreateClient(Factory
+                .WithWebHostBuilder(builder => builder.ConfigureServices(services => services.AddRazorComponents())));
 
             var response = await client.GetAsync("http://localhost/components/routable");
 
             // Assert
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
             var content = await response.Content.ReadAsStringAsync();
 
-            AssertComponent("\n    Router component\n<p>Routed successfully</p>\n", "Routing", content);
+            AssertComponent("\nRouter component\n<p>Routed successfully</p>", "Routing", content);
+        }
+
+        [Fact]
+        public async Task Renders_BasicComponentInteractive_UsingRazorComponents_Prerrenderer()
```

Comment:
Could you replace the 9 instances of `prerrender` in this PR with `prerender`? Thanks :)

Reply:
Done, although I could only find 5